busman/src/main/java/net/bbuzz/busman/ManifestActivity.java

```diff
@@ -240,11 +250,11 @@ protected void onCreate(Bundle savedInstanceState) {
         final Button addButton = (Button) findViewById(R.id.add_to_manifest_button);
         final Button dropButton = (Button) findViewById(R.id.drop_from_manifest_button);
         mLatestRider = (TextView) findViewById(R.id.manifest_latest_rider);
-        mEmptyListView = (TextView) findViewById(android.R.id.empty);
```

Comment:
Btw, you no longer need to cast views in findViewById(), it's now using generics.

Reply:
Sounds good. Thanks